### DISCUSSION TITLE
“There was an error loading notification settings.”

### DISCUSSION THREAD

**Discussion by igor-petrik-invitae** [community/community#7303] [Mobile] [answered]
On iOS, in my notification settings, I see an error: “There was an error loading notification settings. Please check your internet connection and restart the app.” I am on WiFi, and I have restarted the app.

**Comment by brianlovin** [comment]
@igor-petrik-invitae is this error still happening for you? If so, I can make sure our team is able to start investigating what might be going on here.

**Reply by igor-petrik-invitae** [reply]
Yes, this is still occurring.

**Reply by brianlovin** [reply]
Hey @igor-petrik-invitae we looked into our systems and we think the issue might be on the client. I'm sorry for the trouble here, but if you could try re-installing the app and signing in again, that would help us track things down. If the notifications settings view is still throwing an error after this, we can move on to the next steps.

**Reply by igor-petrik-invitae** [reply]
I have logged out, deleted the app, reinstalled the app, and logged in again. I still see the error.

**Reply by brianlovin** [reply]
Okay, thank you for trying that. I've passed this along to our team and we will work on figuring out what could be going wrong here.

**Comment by michaelblind** [comment]
> On iOS, in my notification settings, I see an error: “There was an error loading notification settings. Please check your internet connection and restart the app.” I am on WiFi, and I have restarted the app.

+1 same issue here

**Comment by brianlovin** [comment, marked as answer]
@michaelblind @igor-petrik-invitae — we've just pushed a fix to our API that should _hopefully_ resolve these errors on your notification settings view. Could you give it another try?

**Reply by igor-petrik-invitae** [reply]
Looks good! Thanks!

**Reply by michaelblind** [reply]
Looks great!
